src/explore-education-statistics-frontend/src/prototypes/components/PrototypeSearchForm.tsx

```diff
@@ -86,46 +86,42 @@ class PrototypeSearchForm extends Component<{}, State> {
     const searchResults: SearchResult[] = elements.map(element => {
       let scrollIntoView: () => void;
 
-      const collapsedContainer = PrototypeSearchForm.parentUntilClassname(
+      const accordionContainer = PrototypeSearchForm.parentUntilClassname(
         element,
         'govuk-accordion__section',
+      );
+
+      const detailsContainer = PrototypeSearchForm.parentUntilClassname(
+        element,
         'govuk-details',
       );
 
+      const tabContainer = PrototypeSearchForm.parentUntilClassname(
+        element,
+        'govuk-tabs',
```

Comment:
Rather than hard-coding class names directly inside of this component, it would probably be better to put these inside of the various associated components as either static properties or extra imports. This would make refactoring easier in the future if the class names change.